app/pages/index.js

```diff
@@ -81,8 +89,15 @@ export async function getStaticProps({ params }) {
   const topNavItems = await fetchAPI('/top-nav-item');
   const topPosts = await fetchAPI('/discourses');
 
+  const i = await fetch('https://api.github.com/repos/RocketChat/RC4Community/issues', {
```

Comment:
if we're planning on extending github integration, better use octokit than any http client.

Reply:
I've used octokit now, can you please check?

Reply:
I thought it will add as a new dependency to our application but I think it can be very useful during extension!